locales/en-human.json

```diff
@@ -27,6 +27,14 @@
   "CHARTS": "Charts",
   "MARKETS": "Markets",
   "EXPLORE": "Explore",
```

Comment:
Copy _suggestions_:

"WHATS_NEW_EXPLAIN": "We now have a \"What's New\" section! Check back here to see the changes we've been making to Blockchain Wallet."

"SIGN_VERIFY_EXPLAIN" : "Prove that you own a bitcoin address by signing a message. Prove that someone else owns a bitcoin address by verifying a signature."

"WATCH_ONLY_EXPLAIN" : "Monitor a bitcoin address without importing its private key."